src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc

```diff
@@ -751,7 +751,9 @@ void TransportAdapterImpl::ConnectFailed(const DeviceUID& device_handle,
 void TransportAdapterImpl::RemoveFinalizedConnection(
     const DeviceUID& device_handle, const ApplicationHandle& app_handle) {
   const DeviceUID device_uid = device_handle;
+#if ENABLE_LOG
   const ApplicationHandle app_uid = app_handle;
+#endif
```

Comment:
It looks like `connections_` type `ConnectionMap` is defined as `std::map<std::pair<DeviceUID, ApplicationHandle>, ConnectionInfo>`.  It also looks like all other uses of `connections_.find` with `make_pair` use `app_handle`.
It looks like I could change the following log methods to use `app_handle` in order to make the macro and definition of `app_uid` unnecessary.  What do you think?